Sources/AloeStackView/Protocols/Highlightable.swift

```diff
@@ -13,28 +13,41 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import UIKit
+
 /**
- * Indicates that a row in an `AloeStackView` should change its background color to a
- * highlighted color when a user is touching it.
+ * Indicates that a row in an `AloeStackView` should be highlighted when the user touches it.
  *
  * Rows that are added to an `AloeStackView` can conform to this protocol to have their
- * background color automatically change to a highlighted color when the user is pressing down on
+ * highlightAction(isHighlighting:) will be performed when the user is pressing down on
  * them.
  */
 public protocol Highlightable {
 
-  /// Checked when the user touches down on a row to determine whether the background color of the
-  /// row should be changed to a highlighted color.
+  /// Checked when the user touches down on a row to determine if the row should be highlighted.
   ///
   /// The default implementation of this method always returns `true`.
   var isHighlightable: Bool { get }
 
+  /// Called when the highlighted state of the row changes.
+  /// Override this method to provide custom highlighting behavior for the row.
+  ///
+  /// The default implementation of this method changes the background color of the row to the `rowHighlightColor`.
+  func setIsHighlighted(_ isHighlighted: Bool)
 }
 
-extension Highlightable {
+extension Highlightable where Self: UIView {
 
   public var isHighlightable: Bool {
-    get { return true }
+    return true
   }
 
+  public func setIsHighlighted(_ isHighlighted: Bool) {
+    guard let cell = superview as? StackViewCell else { return }
+    if isHighlighted {
+      cell.backgroundColor = cell.rowHighlightColor
```

Comment:
Minor: make this a ternary if it fits on one line:

    cell.backgroundColor = isHighlighted ? cell.rowHighlightColor : cell.rowBackgroundColor

If it doesn’t fit on one line, it’s ok to leave it as it is.